### PR TITLE
Adding some more style

### PR DESCRIPTION
**Pull request by toddtarsi**
This was concerned with making the ux responsive to dark/light mode
as well as various window sizes

### CODE REVIEW

File: theme/p2/base.p2 [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   <head>
     <title>{% block title %}Yurt Tools{% endblock %}</title>
     <link href="/static/css/reset.css" rel="stylesheet">
     <link href="/static/css/theme.css" rel="stylesheet">
   </head>
   <body>
-    <div id="top-bar" class="bar">
-      <div class="bar-item">Yurt Tools</div>
+    <div class="nav">
+      <h1>Yurt Tools</h1>
```
Comment (the-maldridge) [inline theme/p2/base.p2:10]: Lets make this more specific.  How about `Task Info` and we can eventually have a footer somewhere to put yurt-tools in as a pointer to here.

File: theme/static/css/theme.css [hunk 2]
```diff
@@ -1,59 +1,113 @@
...
+        color: #121212;
+    }
+    .card {
+        background: var(--darken-color);
+        border: 1px solid var(--darken-color);
+    }
+    .nav {
+        background: var(--darken-color);
+    }
+}
```
Comment (the-maldridge) [inline theme/static/css/theme.css:113]: Missing newline.

File: theme/p2/taskinfo.p2 [hunk 3]
```diff
@@ -3,25 +3,31 @@
...
-          </ul>
-        </li>
+    <div class="card grid">
+      <h3>{{job}}</h3>
+      {% for group, tasks in groups %}
+      <div class="card grid">
+        <h4>{{group}}</h4>
+        {% for task, info in tasks %}
+        <div class="grid m-3">
+          {{task}} -
```
Comment (the-maldridge) [inline theme/p2/taskinfo.p2:17]: So for each of the various info sources (version is the only one being loaded right now, but there are others that taskinfo knows about) my intent is to display an icon for each: grey if no data is available, green if the status is good, and red if the status is bad.  I think these icons should stack up along the right hand side of each task block, can you make the change to accommodate this?

**Review by the-maldridge — CHANGES_REQUESTED** [review]

File: theme/p2/taskinfo.p2 (hunk 3 above)
Comment (toddtarsi) [reply theme/p2/taskinfo.p2:17]: @the-maldridge - Yessir, do you mean similar to the icons on master?

Screen Shot 2021-12-02 at 10 40 27 PM

Comment (the-maldridge) [reply theme/p2/taskinfo.p2:17]: Similar feel would be nice, no need to try and match them exactly.

Comment (toddtarsi) [reply theme/p2/taskinfo.p2:17]: @the-maldridge - I think I might need some hand holding here just to figure out what iteration over the info sources looks like if thats okay. This data shape feels very bespoke and my go-chops are... formative.

Comment (the-maldridge) [reply theme/p2/taskinfo.p2:17]: You are correct, the shapes are completely bespoke based on what info provider has sourced the information.  The base structure though is a map and so you can use that to decide whether the icon should be grey or not.  Then each one should have a variable that indicates that it is in the "good" state.  For the version checker this is "UpToDate".  For trivy I don't remember and I'll have to check.

**Comment by toddtarsi** [comment]
@the-maldridge - I believe all requested changes are done here now. Svgs are rendered inline (this is easiest to work with). 

Current UX:

Dark:
Screen Shot 2021-12-11 at 7 20 28 PM

Light:
Screen Shot 2021-12-11 at 7 24 30 PM

**Comment by the-maldridge** [comment]
Did you flip the layout around yet?  I remember the last time we spoke the only feedback I had remaining was to flip it around so it went labels then icons with icons right aligned in the box stacking to the left.  Once you make this change I'm good to merge.

Please squash your commits and write a meaningful commit message.

**Comment by the-maldridge** [comment]
Oh, and I just realized you're using the lock icon for update status.  Please swap this out for the clock one.  Lets reserve the lock one for security status.

**Comment by toddtarsi** [comment]
Dark mode:

Screen Shot 2021-12-17 at 1 21 44 AM

Task info selected (will be shrank to sidebar when main tab is opened:

Screen Shot 2021-12-17 at 1 24 22 AM

Light mode:

Screen Shot 2021-12-17 at 1 26 11 AM

**Pull request merged**